### PR TITLE
Add super-supplier account credentials

### PR DESCRIPTION
**Pull request by paololim**
Add super supplier account credentials.

@blueapron/wine

### CODE REVIEW

**Comment by dylandrop** [comment]
:+1:

File: lib/ship_compliant/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module ShipCompliant
-  VERSION = "0.2.1"
```
Comment (nikhilofthesouth) [inline lib/ship_compliant/version.rb:2]: NB: This should be a minor version bump

Comment (paololim) [inline lib/ship_compliant/version.rb:2]: 0.3.0?

**Comment by nikhilofthesouth** [comment]
:+1:

File: lib/ship_compliant/version.rb (hunk 1 above)
Comment (nikhilofthesouth) [inline lib/ship_compliant/version.rb:2]: Yep. For reference if you've never seen it before: http://semver.org/ is a very commonly accepted convention

**Pull request merged**